config/config-example.yml

```diff
@@ -19,17 +19,17 @@ photo_path: "/example/path/to/photo/folder"
 # separate calibration per path) and that align_photos:reset_alignment must be False and
 # align_photos:keep_keypoints must be True. If there are no secondary photos to add, set to an empty
 # string ("").
-photo_path_secondary: "/example/path/to/secondary/photo/folder"
+photo_path_secondary: ""
 
 # Path for exports (e.g., points, DSM, orthomosaic) and processing log. Will be created if does not exist.
-output_path: "/example/output/path"
+output_path: "/home/asidhu/automate-metashape-2/initialRun"
 
 # Path to save Metashape project file (.psx). Will be created if does not exist
-project_path: "/example/project/path"
+project_path: "/home/asidhu/automate-metashape-2/initialRun"
 
 # The identifier for the run. Will be used in naming output files. Recommended to include a photoset name and processing parameter set name.
 # Optionally, set it to an empty string ("") to use the config file name (minus extension) as the run name
-run_name: "example-run-001"
+run_name: "emeraldData-run-001"
```

Comment:
Same comment about examples.

Reply:
I don't think you should commit any changes to this config.

Reply:
I think I ran `git add .` and so that unintentionally added the config file also. I agree not to commit changes to the config.